e2e/stories/select-record.e2e.ts

```diff
@@ -0,0 +1,371 @@
+import * as faker from "faker";
+import * as Joi from "joi";
+import { field } from "../../src";
+import { cleaning, DEFAULT_DATASET, dom, init } from "../teardowns";
+import { MESSAGE } from "./../../src/config/message";
+
+const joiAssert = Joi.assert;
+
+jest.setTimeout(15000); // for unstable internet connection
+
+describe("filter records REST API", async () => {
+  const getDataSet = () => dom.dataset(DEFAULT_DATASET.NAME);
+  const BAD_REQUEST = new Error(`${MESSAGE.CORE.BACKEND_ERROR}400 Bad Request`);
+
+  const FIELD = {
+    BOOLEAN: "boolean_field",
+    INTEGER: "integer_field",
+    FLOAT: "float_field",
+    STRING: "string_field",
+  };
+
+  function testLength(title: string, where, expectedLength: number) {
```

Comment:
What do you think if we change `where` to `condition`? In order to avoid that weird `where(where)`